deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
```

Comment:
It would be good to move all of the files that we made related to deployment, running test environments etc into `bin` to keep the root directory tidy.

Reply:
Sure good idea. I moved most of the test/deploy files into bin but the 3 Dockerfiles need to stay in root as docker context can't move to parent directories.